tracer/test/snapshots/NUnitEvpTests.SubmitTraces_packageVersion=all_evpVersionToRemove=null_expectedGzip=null.verified.txt

```diff
@@ -2028,6 +2029,9 @@
       ci.stage.name: StageDisplayName,
       ci.workspace_path: workspacePath,
       env: integration_tests,
+      error.msg: ErrorMessage,
+      error.stack: StackTrace,
+      error.type: System.Exception,
```

Comment:
Again, is this an unrelated bug fix?

Reply:
Yes, I was missing those tags for the exception in this case.